.circleci/config.yml

```diff
@@ -1,8 +1,19 @@
 version: 2.1
 
 orbs:
-  node: circleci/node@4.7
-  win: circleci/windows@2.4.1
+  win: circleci/windows@4.1.1
+
+executors:
+  windows: &windows
+    machine:
+      image: windows-server-2022-gui:edge
```

Comment:
Why are we using the `edge` variant?

Reply:
Also CircleCI strongly recommend using orbs instead of configuring machine directly, so I think we should stick to orbs. https://circleci.com/docs/using-windows

Reply:
For Using Orbs, although it is technically possible to use orbs with matrix jobs, but because we can't set parameter names for orbs (i.e. &windows, &macos), matrix itself and OS conditional statements really gets ugly. so I had to use machine directly. https://circleci.com/docs/using-matrix-jobs

Reply:
And for the `edge` variant, If I remember correctly `current` variant's preinstalled .NET SDK version stays on 5, so In order to utilize preinstalled SDK to minimize installation time, I used `edge` variant. 

But on the other hand, I'm not sure about it, so I'll try the `current` variant shortly after.

Reply:
Good news for ya, .NET 6 is available on `current` as well: https://discuss.circleci.com/t/june-2022-windows-updates-available-for-current-tags/44540

Reply:
Confirmed, using `current`.